src/ints/int10_memory.cpp

```diff
@@ -22,112 +22,179 @@
 #include "mem.h"
 #include "inout.h"
 
-static uint8_t static_functionality[0x10]=
-{
- /* 0 */ 0xff,  // All modes supported #1
- /* 1 */ 0xff,  // All modes supported #2
- /* 2 */ 0x0f,  // All modes supported #3
- /* 3 */ 0x00, 0x00, 0x00, 0x00,  // reserved
- /* 7 */ 0x07,  // 200, 350, 400 scan lines
- /* 8 */ 0x04,  // total number of character blocks available in text modes
- /* 9 */ 0x02,  // maximum number of active character blocks in text modes
- /* a */ 0xff,  // Misc Flags Everthing supported 
- /* b */ 0x0e,  // Support for Display combination, intensity/blinking and video state saving/restoring
- /* c */ 0x00,  // reserved
- /* d */ 0x00,  // reserved
- /* e */ 0x00,  // Change to add new functions
- /* f */ 0x00   // reserved
+// clang-format off
+static uint8_t static_functionality[0x10] = {
+    /* 0 */ 0xff,  // All modes supported #1
+    /* 1 */ 0xff,  // All modes supported #2
+    /* 2 */ 0x0f,  // All modes supported #3
+    /* 3 */ 0x00, 0x00, 0x00, 0x00,  // reserved
+    /* 7 */ 0x07,  // 200, 350, 400 scan lines
+    /* 8 */ 0x04,  // total number of character blocks available in text modes
+    /* 9 */ 0x02,  // maximum number of active character blocks in text modes
+    /* a */ 0xff,  // Misc Flags Everthing supported 
+    /* b */ 0x0e,  // Support for Display combination, intensity/blinking and video state saving/restoring
+    /* c */ 0x00,  // reserved
+    /* d */ 0x00,  // reserved
+    /* e */ 0x00,  // Change to add new functions
+    /* f */ 0x00   // reserved
 };
 
-static uint16_t map_offset[8]={
-	0x0000,0x4000,0x8000,0xc000,
-	0x2000,0x6000,0xa000,0xe000
+static std::vector<uint16_t> map_offset = {
+	0x0000, 0x4000, 0x8000, 0xc000,
+	0x2000, 0x6000, 0xa000, 0xe000
 };
+// clang-format on
+
+void INT10_LoadFont(const PhysPt _font, const bool reload, const int count,
+                    const int offset, const int map, const int height)
+{
+	PhysPt ftwhere = PhysicalMake(0xa000,
+	                              map_offset[map & 0x7] +
+	                                      (uint16_t)(offset * 32));
+
+	uint16_t base = real_readw(BIOSMEM_SEG, BIOSMEM_CRTC_ADDRESS);
+
+	bool mono = (base == VGAREG_MDA_CRTC_ADDRESS);
+
+	// Put video adapter in planar mode
+	IO_Write(0x3c4, 0x02);
+
+	// select plane 2 for writing
+	IO_Write(0x3c5, 0x04);
```

Comment:
Formatting here is borked 🐔 . The comments are misplaced. I would expect something like:
```
// Put video adapter in planar mode

// select plane 2 for writing
IO_Write(0x3c4, 0x02);
IO_Write(0x3c5, 0x04);

// odd/even off in SEQ
IO_Write(0x3c4, 0x04);
IO_Write(0x3c5, 0x07);

...
```